public/pages/ReviewAndCreate/components/ModelConfigurationFields/ModelConfigurationFields.tsx

```diff
@@ -188,8 +202,92 @@ export const ModelConfigurationFields = (
     };
   };
 
+  const handleFeatureAttributesCallout = (
+    issueResponse: validationModelResponse
+  ) => {
+    if (issueResponse != undefined && issueResponse != null) {
```

Comment:
Removed null check. I used to need the null check when I defined the interface for `ValidationModelResponse` as either an `object` or `null` but now I use `undefined`. I mistakenly just added `undefined` check instead of removed the not `null` check.